### DISCUSSION TITLE
Error: no case clause matching: [] in Plausible.Slack.notify/1

### DISCUSSION THREAD

**Discussion by albertmoravec** [plausible/analytics#918] [Self-Hosted Support] [answered]
### Precheck

Please note that this tracker is only for bugs. Do not use the issue tracker for help, support or feature requests. 

[Our docs](https://plausible.io/docs) are a great place for most answers, but if you can’t find your answer there, you can [contact us](https://plausible.io/contact). 

Have a feature request? Please search the ideas [on our forum](https://github.com/plausible/analytics/discussions/categories/feature-requests) to make sure that the feature has not yet been requested. If you cannot find what you had in mind, please [submit your feature request here](https://github.com/plausible/analytics/discussions/new). 

Have an issue with your self-hosted install? You can ask in [our self-hosted forum](https://github.com/plausible/analytics/discussions/categories/self-hosted-support). Thanks!

## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report
**Describe the bug**
Upon creating a new self-hosted instance and going through initial setup I got the following error logged in console

```
23:32:36.434 [error] Task #PID<0.3621.0> started from #PID<0.3620.0> terminating
** (CaseClauseError) no case clause matching: []
    (hackney 1.17.0) /app/deps/hackney/src/hackney_url.erl:248: :hackney_url.parse_netloc/2
    (hackney 1.17.0) /app/deps/hackney/src/hackney.erl:351: :hackney.request/5
    (httpoison 1.8.0) lib/httpoison/base.ex:846: HTTPoison.Base.request/6
    (httpoison 1.8.0) lib/httpoison.ex:156: HTTPoison.request!/5
    (elixir 1.11.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: #Function<0.34765851/0 in Plausible.Slack.notify/1>
    Args: []
```

It was not a fatal error and I couldn't see anything on the frontend, so I'm not sure when exactly during the initialization this error happened.

**Environment (If applicable):**
 - OS: Linux (Centos 8)

**Comment by julienma** [comment]
Same error, which seems to happen only when I use the [custom clickhouse config files](https://github.com/plausible/hosting/tree/master/clickhouse). If I remove these 2 custom .xml and restart clickhouse, I don't have this error anymore in the log.

**Comment by krutilin** [comment]
Almost the same issue, but after each error service stoped. As @julienma suggested I commented lines with these files in docker-compose.yml and I still have an error in the console.

I receive this error each time on adding the site to the analytics.

```
13:39:58.586 [error] Task #PID<0.4209.0> started from #PID<0.4208.0> terminating
** (CaseClauseError) no case clause matching: []
    (hackney 1.17.4) /app/deps/hackney/src/hackney_url.erl:248: :hackney_url.parse_netloc/2
    (hackney 1.17.4) /app/deps/hackney/src/hackney.erl:351: :hackney.request/5
    (httpoison 1.8.0) lib/httpoison/base.ex:846: HTTPoison.Base.request/6
    (httpoison 1.8.0) lib/httpoison.ex:156: HTTPoison.request!/5
    (elixir 1.11.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: #Function<0.34765851/0 in Plausible.Slack.notify/1>
    Args: []
```

**Comment by ghost** [comment, marked as answer]
Confirming this issue on a fresh setup.
```
Debian GNU/Linux 11
Kernel version 5.10.0-9-amd64
Docker version 20.10.10, build b485636
docker-compose version 1.29.2, build 5becea4c
```

Some containers infos:
```
$ sudo docker ps -a
CONTAINER ID   IMAGE                               COMMAND                  CREATED         STATUS         PORTS                                       NAMES
ff7bfdbc6f6a   plausible/analytics:latest          "/entrypoint.sh sh -…"   6 minutes ago   Up 6 minutes   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   hosting_plausible_1
5c8b70341480   bytemark/smtp                       "docker-entrypoint.s…"   6 minutes ago   Up 6 minutes   25/tcp                                      hosting_mail_1
570ac9f0802c   yandex/clickhouse-server:21.3.2.5   "/entrypoint.sh"         6 minutes ago   Up 6 minutes   8123/tcp, 9000/tcp, 9009/tcp                hosting_plausible_events_db_1
8958df3afff0   maxmindinc/geoipupdate              "/usr/bin/entry.sh"      6 minutes ago   Up 6 minutes                                               hosting_geoip_1
783c79e97f15   postgres:12                         "docker-entrypoint.s…"   6 minutes ago   Up 6 minutes   5432/tcp
```

**Reply by ukutaht** [reply]
Can you try pulling the `latest` tag again and seeing if this is still an issue? I attempted to fix it in `v1.4.0` but I hadn't updated the `latest` tag until today

**Reply by ghost** [reply]
Thanks, I updated the container and the issue disappeared.

**Reply by krutilin** [reply]
Thanks, fixed after update too.
